NEWS.md

```diff
@@ -86,6 +86,7 @@ New library functions
 New library features
 --------------------
 
+* `redirect_*`, functions now accept `devnull` to discard all output redirected to it, and as an empty input. ([#36146]).
```

Comment:
To make it the same as the one below:

```suggestion
* The `redirect_*` functions now accept `devnull` to discard all output redirected to it, and as an empty input. ([#36146]).
```

but I don't understand "and as an empty input".

Reply:
It is empty when you read it.

```
julia> redirect_stdin(devnull) do
       println("*", readline(), "*")
       end
**
```